engine/common/src/main/java/org/enso/common/ContextFactory.java

```diff
@@ -197,7 +196,7 @@ PolyglotContext build() {
           .option("java.UseBindingsLoader", "true")
```

Comment:
This `if (ENGINE_HAS_JAVA)` block is now shared between CLI and language-server.